global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "rollForward": "latestMajor",
-    "version": "6.0.200"
+    "rollForward": "latestFeature",
```

Comment:
we only want features, we don't want to automatically roll to 8.0 later on. We'll stay on 7.